src/pages/RepoPage/FailedTestsTab/FailedTestsPage/FailedTestsTable/FailedTestsTable.tsx

```diff
@@ -84,44 +99,74 @@ export function getSortingOption(
 const isNumericValue = (value: string) =>
   value === 'avgDuration' ||
   value === 'failureRate' ||
-  value === 'commitsFailed'
+  value === 'commitsFailed' ||
+  value === 'flakeRate'
 
 interface FailedTestsColumns {
   name: string
   avgDuration: number | null
   failureRate: number | null
+  flakeRate?: React.ReactNode
   commitsFailed: number | null
   updatedAt: string
 }
 
 const columnHelper = createColumnHelper<FailedTestsColumns>()
 
-const columns = [
-  columnHelper.accessor('name', {
-    header: () => 'Test name',
-    cell: (info) => info.renderValue(),
-  }),
-  columnHelper.accessor('avgDuration', {
-    header: () => 'Average duration',
-    cell: (info) => `${(info.renderValue() ?? 0).toFixed(3)}s`,
-  }),
-  columnHelper.accessor('failureRate', {
-    header: () => 'Failure rate',
-    cell: (info) => {
-      const value = (info.renderValue() ?? 0) * 100
-      const isInt = Number.isInteger(info.renderValue())
-      return isInt ? `${value}%` : `${value.toFixed(2)}%`
-    },
-  }),
-  columnHelper.accessor('commitsFailed', {
-    header: () => 'Commits failed',
-    cell: (info) => (info.renderValue() ? info.renderValue() : 0),
-  }),
-  columnHelper.accessor('updatedAt', {
-    header: () => 'Last run',
-    cell: (info) => formatTimeToNow(info.renderValue()),
-  }),
-]
+const getColumns = ({
+  hideFlakeRate,
+  interval,
+}: {
+  hideFlakeRate: boolean
+  interval?: MeasurementInterval
+}) => {
+  const baseColumns = [
+    columnHelper.accessor('name', {
+      header: () => 'Test name',
+      cell: (info) => info.renderValue(),
+    }),
+    columnHelper.accessor('avgDuration', {
+      header: () => 'Avg duration',
+      cell: (info) => `${(info.renderValue() ?? 0).toFixed(3)}s`,
+    }),
+    columnHelper.accessor('failureRate', {
+      header: () => 'Failure rate',
+      cell: (info) => {
+        const value = (info.renderValue() ?? 0) * 100
+        const isInt = Number.isInteger(info.renderValue())
+        return isInt ? `${value}%` : `${value.toFixed(2)}%`
```

Comment:
Technically if info.renderValue() is not a number this could end up saying `NaN%`. Maybe an edge case don't need to expect or can add to post-release polish as needed

Reply:
For the failure rate (the value here), the API always returns a float, so we're safe!
ref: https://github.com/codecov/codecov-api/blob/main/graphql_api/types/test_results/test_results.graphql#L5